File: Napakalaki Java/NapakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author antoniomj
 */
public class Prize {
    
    private  int level;
    private  int treasures;
    
    public Prize(int Treasures, int Level){
        
        treasures =Treasures;
        //Examen
        if(treasures > 5)
            treasures = 5;
        //Fin examen
        level = Level;
                
    } 
    
    public int GetTreasures(){
        
        return treasures;
    }
    
    public int Getlevel(){
        
        return level;
    }
    
    
    public String toString(){
        
        return "Nivel del premio: " + Integer.toString(level) + 
                "\nNumero de tesoros: " + Integer.toString(treasures);
           
    }
}

File: Napakalaki Ruby/Prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module NapakalakiGame

class Prize
  
  attr_reader :treasures, :level;
  
  def initialize(tesoros, niveles)
    
    @treasures = tesoros
    @level = niveles
    
  end
  
  def to_s ()
    
    return "El premio por matar este monstruo es: \nTesoros ganados: #{@treasures}\nNiveles Ganados: #{@level}"
    
  end
  
  def getTreasures
    return @treasures
  end
  
  def getLevels
    return @level
  end
end

end
